src/pretalx/orga/forms/submission.py

```diff
@@ -181,6 +197,17 @@ def save(self, *args, **kwargs):
             task_update_unreleased_schedule_changes.apply_async(
                 kwargs={"event": self.event.slug}
             )
+        if (
+            instance.state in SubmissionStates.accepted_states
+            and not self.cleaned_data.get("room")
+            and not self.cleaned_data.get("start")
```

Comment:
As the form only passes validation if both `room` and `start` are present (or neither), checking for one here should be enough.